source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+* Sii conscio dell' ambiente e dei suoi partecipanti. Se noti una situazione pericolosa, qualcuno che non si sente a suo agio, o una violazione di questo Codice di Condotta, avvisa immediatamente i leaders della community, anche se la situazione non sembra poter avere conseguenze particolarmente dannose.
+
+
+Comportamento inaccettabile
+---------------------
+
+I comportamenti che riteniamo inaccettabili includono: intimidatori, molesti, abusanti, discriminatori, linguaggi o azioni dispregiativi e avvilenti perpetuati da qualsiasi partecipante nella nostra community online, in tutti gli eventi collegati e nelle conversazioni tra singoli pertinenti agli interessi della community. Alcuni eventi della Community potrebbero aver luogo in spazi condivisi con un pubblico esterno alla community stessa; per favore, sii rispettoso/a con tutti/e i/le promotori/e dei suddetti spazi.
+
+Comportamenti molesti includono: commenti scritti o verbali dannosi o pregiudizievoli in relazione al sesso, orientamento sessuale, razza, religione, disabilità fisiche e/o mentali; uso inappropriato di immagini di nudità e/o immagini a sfondo sessuale utilizzate in spazi pubblici (comprese le slides per le presentazioni); intimidazioni, il seguire fisicamente o stalking; molestie registrate sotto forma di fotografie e video; interruzione sostenuta di presentazioni o altri eventi; contatto fisico inappropriato, e sgradita attenzione sessuale.
+
+
+Conseguenze per i comportamenti inaccettabili
+-------------------------------------
+
+Qualsiasi comportamento ritenuto inaccettabile perpetuato da qualsiasi membro della community, compresi Sponsors e coloro nella posizione di potere decisionale, non sarà tollerato.
+Ci aspettiamo che chiunque venga ripreso nell' agire in maniera molesta, interrompa le suddette azioni immediatamente.
+
+Se un membro della community agisce in maniera non consona al Codice di Condotta, gli organizzatori della community possono reagire di conseguenza nel modo che più ritengono appropriato, incluso escludere temporaneamente o permanentemente il membro dalla community, senza preavviso alcuno (e senza rimborso in caso di evento a pagamento).
+
+
+Se noti o sei soggetto/a di un comportamento inaccettabile
+------------------------------------------------------
+
+Nel caso notassi o fossi soggetto/a a un comportamento inaccettabile, o avessi bisogno di chiarimenti, notifica uno/a degli/le organizzatori/ci il prima possibile.
+Puoi trovare una lista di organizzatori/ci da contattare per ognuno dei sostenitori di questo codice di condotta, alla fine di questa pagina.
+Inoltre, gli organizzatori/ci della community sono disponibili nell' aiutare i membri della community nel contattare le autorita' giudiziarie locali o rassicurare e ripristinare un livello di sicurezza adeguato per coloro bersaglio di comportamenti inaccettabili . In caso di eventi in persona, gli/le organizzatori/ci, nel caso colui/lei lo desideri, possono provvedere ad una scorta.
```

Comment:
1. no space between the apostrophe and the following word.
2. also, please use "à" and not " a' " (and similar). apostrophe and stress are different.
3. "per coloro che sono oggetto"
4. no space before the period "inaccettabili." 
5. "in caso di eventi che avvengano di persona". This is really difficult, I can't find a nicer way to put it.